src/Symfony/Bridge/Doctrine/Security/User/EntityUserProvider.php

```diff
@@ -114,7 +118,7 @@ private function getObjectManager()
 
     private function getRepository()
     {
-        return $this->getObjectManager()->getRepository($this->classOrAlias);
+        return (null !== $this->repository) ? $this->repository : $this->getObjectManager()->getRepository($this->classOrAlias);
```

Comment:
`return $this->repository ?? $this->getObjectManager()->...;`

Reply:
I wondering why not `?:` instead of `??` ? Is not always `$this->repository` a defined property?

Reply:
@yceruto `??` still applies on `null` on defined props :)

Reply:
Yeah `?:` too :) that's the question: it shouldn't be `??` *only* used for cases where the *var* can be undefined?

Reply:
Hm. Touche :) I'd say ?? is more pure and scales better, as we only anticipate `null`. ?: is basically empty() in disguise.

Reply:
I get it! 1 - 0 :)